pyanaconda/modules/storage/zfcp/zfcp.py

```diff
@@ -84,3 +84,15 @@ def process_kickstart(self, data):
     def setup_kickstart(self, data):
         """Setup the kickstart data."""
         data.zfcp.zfcp = self._zfcp_data
+        # So far, the data contains explicit zfcp statements from a
+        # kickstart file used as input for the installation. Now, add any
+        # missing entries that come from user interaction with the GUI.
+        for fcpdev in zfcp.fcpdevs:
+            zd = data.zfcp.dataClass()
+            zd.devnum = fcpdev.devnum
+            if "wwpn" in dir(fcpdev):
+                zd.wwpn = fcpdev.wwpn
```

Comment:
This code is ported from upstream, isn't it? It should be ideally `zd.wwpn = getattr(fcpdev, "wwpn", "")` and same for `fcplun`, but it doesn't make sense to change the original commits here.

Reply:
I think we can change it here, I'll open also a PR for upstream.